rego/Drive.rego

```diff
@@ -216,34 +354,57 @@ tests contains {
 }
 if {
     DefaultSafe := false
-    Events := utils.FilterEvents(LogEvents, "PUBLISHING_TO_WEB", utils.TopLevelOU)
+    Events := utils.FilterEventsOU(LogEvents, "PUBLISHING_TO_WEB", utils.TopLevelOU)
     count(Events) == 0
 }
 
 tests contains {
     "PolicyId": "GWS.DRIVEDOCS.1.5v0.1",
     "Criticality": "Shall",
-    "ReportDetails": utils.ReportDetailsOUs(NonCompliantOUs1_5),
-    "ActualValue": {"NonCompliantOUs": NonCompliantOUs1_5},
+    "ReportDetails": utils.ReportDetails(NonCompliantOUs1_5, NonCompliantGroups1_5),
+    "ActualValue": {"NonCompliantOUs": NonCompliantOUs1_5,
+        "NonCompliantGroups": NonCompliantGroups1_5},
     "RequirementMet": Status,
     "NoSuchEvent": false
 }
 if {
-    Events := utils.FilterEvents(LogEvents, "PUBLISHING_TO_WEB", utils.TopLevelOU)
+    Events := utils.FilterEventsOU(LogEvents, "PUBLISHING_TO_WEB", utils.TopLevelOU)
     count(Events) > 0
-    Status := count(NonCompliantOUs1_5) == 0
+    Conditions := {count(NonCompliantOUs1_5) == 0, count(NonCompliantGroups1_5) == 0 }
+    Status := (false in Conditions) == false
 }
 #--
 
 #
 # Baseline GWS.DRIVEDOCS.1.6v0.1
 #--
-NonCompliantOUs1_6 contains OU if {
+GetFriendlyValue1_6(Value):= "Access Checking is disabled outside of docs and drive"
```

Comment:
Access checker isn't an enable/disable setting - in fact it can't be disabled. See https://github.com/cisagov/ScubaGoggles/issues/302.

What's really happening in the non-compliant case is access checker is allowed to suggest the user to that access be granted to all, even non-Google users. 

```suggestion
GetFriendlyValue1_6(Value) := "Access Checker allows users to share files to the public (no Google account required)."
```

Reply:
so friendly method will look like this, no else condition:

![image](https://github.com/cisagov/ScubaGoggles/assets/108771509/a64581e6-e6f1-4148-9c52-bc68097f5a4d)

Reply:
If you get rid of the else, might as well get rid of the function entirely, like so:
![image](https://github.com/cisagov/ScubaGoggles/assets/106177711/c20080b6-3c1e-4d2b-b8c9-1d615377d95b)

Just be sure to update the Groups version of the rule as well.